salt/_modules/caasp_orch.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+
+
+def __virtual__():
+    return "caasp_orch"
+
+
+def sync_all():
+    '''
+    Syncronize everything before starting a new orchestration
+    '''
+    __utils__['caasp_log.debug']('orch: synchronizing the mine')
+    __salt__['saltutil.runner']('mine.update', tgt='*', clear=True)
```

Comment:
can we do here:
```python
     try:
    __salt__['saltutil.runner']('mine.update', tgt='*', clear=True)
            except Exception as exc:
                WARN: YOUR BANK ACCOUNT HAS BEEN HACKED 

```

Reply:
I think there is no need to catch exceptions in every single line of our code. The standard exception handler is fine here.